student.rb

```diff
@@ -1,5 +1,8 @@
 require './person'
```

Comment:
### Optional
- [ ] It would be nice if we use `require_relative` when working with files in our current directory, use `require` when working with packages in our gemfile

Reply:
Hi @addod19, thanks for the review. All suggestions will be implemented